src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyStore.tsx

```diff
@@ -132,55 +132,42 @@ const KeyStoreFlow = () => {
     }
   };
 
+  const sendTagManagerEvent = (category: string, action: string, label: string) => {
+    GoogleTagManager.getInstance().sendEvent({ category, action, label });};
+
   const RemoveButton = () => <Grid ref={removeButtons} onClick={removeFile} className={classes.Remove}>Remove</Grid>;
 
   const submitHandler = async () => {
     applicationStore.setIsLoading(true);
     setTimeout(async () => {
       try {
-      await validatorStore.extractKeyStoreData(keyStorePassword);
-      isDeposited;
-      const deposited = true; // await isDeposited();
-      applicationStore.setIsLoading(false);
-      validatorStore.registrationMode = 1;
-      slashingWarningNavigate[`${processStore.secondRegistration}`]();
+        await validatorStore.extractKeyStoreData(keyStorePassword);
+        // TODO fix this dummy value.
+        const deposited = true; // await isDeposited();
```

Comment:
Unfortunately It was like that. because isDeposited isn't implemented and linter scream that isDeposited isn't used. 'isDeposited;' was just added in the code and 'deposited' was given a dummy value. I added a TODO so we can find it later. I don't want to start implementing the logic of isDeposited in the context of this PR.